app/src/main/java/org/standardnotes/notes/LoginActivity.kt

```diff
@@ -57,7 +57,7 @@ class LoginActivity : AppCompatActivity() {
                 hideProgress()
             }
 
-            override fun onFailure(call: Call<SigninResponse>, t: Throwable) {
+            override fun onFailure(call: Call<SigninResponse>?, t: Throwable?) {
```

Comment:
Ya it probably is..I'm honestly so unfamiliar with my way around this code and Android dev in general, so I kept the first thing that worked. Perhaps at some point someone with more experience could submit a PR that improves this.